src/util/log.hpp

```diff
@@ -5,6 +5,7 @@
 #define LOG_CALLBACK
```

Comment:
I'm not a fan of this design change. I agree that this functionality would be handy but preferably I'd use a preprocessor definition instead of a boolean.